docs/sources/review/cla-assistant/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: About CLA assistant
+description: Understand the CLA assistant and its use on Github Pull Requests.
```

Comment:
GitHub should always be spelled with a capital H and pull requests never needs to be capitalized.

```suggestion
description: Understand the CLA assistant and its use on GitHub pull requests.
```